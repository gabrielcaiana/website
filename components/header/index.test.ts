import { describe, it, expect } from 'vitest'
import { mount } from '@vue/test-utils'
import Header from './index.vue'

describe('Header', () => {
  it('should render Header', () => {
    const wrapper = mount(Header)
    expect(wrapper.exists()).toBeTruthy()
    expect(wrapper).toMatchSnapshot()
  })
})
