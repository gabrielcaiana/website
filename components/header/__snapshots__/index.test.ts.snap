// Vitest Snapshot v1

exports[`Header > should render Header 1`] = `
VueWrapper {
  "__app": {
    "_component": {
      "name": "VTU_ROOT",
      "render": [Function],
      "setup": [Function],
    },
    "_container": <div
      data-v-app=""
    >
      <header
        class="pt-8 pb-4 px-6 text-white border-b border-neutral-800 flex justify-between items-center"
      >
        <nuxt-link
          class="font-medium"
          to="/"
        >
          Gabriel Caiana
        </nuxt-link>
        <div
          class="flex gap-3"
        >
          <a
            href="https://github.com/gabrielcaiana"
            target="_blank"
          >
            <icon
              name="ph:github-logo-fill"
              size="24"
            />
          </a>
          <a
            href="https://twitter.com/gabriel_caiana"
            target="_blank"
          >
            <icon
              name="ph:twitter-logo-fill"
              size="24"
            />
          </a>
          <a
            href="https://www.linkedin.com/in/gabrielcaiana/"
            target="_blank"
          >
            <icon
              name="ph:linkedin-logo-fill"
              size="24"
            />
          </a>
        </div>
      </header>
    </div>,
    "_context": {
      "app": [Circular],
      "components": {
        "transition": {
          "name": "transition",
        },
        "transition-group": {
          "name": "transition-group",
        },
      },
      "config": {
        "compilerOptions": {},
        "errorHandler": undefined,
        "globalProperties": {},
        "isNativeTag": [Function],
        "optionMergeStrategies": {},
        "performance": false,
        "warnHandler": undefined,
      },
      "directives": {},
      "emitsCache": WeakMap {},
      "mixins": [
        {
          "beforeCreate": [Function],
        },
      ],
      "optionsCache": WeakMap {},
      "propsCache": WeakMap {},
      "provides": {},
      "reload": [Function],
    },
    "_instance": {
      "a": null,
      "accessCache": {},
      "appContext": {
        "app": [Circular],
        "components": {
          "transition": {
            "name": "transition",
          },
          "transition-group": {
            "name": "transition-group",
          },
        },
        "config": {
          "compilerOptions": {},
          "errorHandler": undefined,
          "globalProperties": {},
          "isNativeTag": [Function],
          "optionMergeStrategies": {},
          "performance": false,
          "warnHandler": undefined,
        },
        "directives": {},
        "emitsCache": WeakMap {},
        "mixins": [
          {
            "beforeCreate": [Function],
          },
        ],
        "optionsCache": WeakMap {},
        "propsCache": WeakMap {},
        "provides": {},
        "reload": [Function],
      },
      "asyncDep": null,
      "asyncResolved": false,
      "attrs": {},
      "bc": null,
      "bm": null,
      "bu": null,
      "bum": null,
      "c": null,
      "components": null,
      "ctx": {
        "VTU_COMPONENT": {},
      },
      "da": null,
      "data": {},
      "devtoolsRawSetupState": {
        "VTU_COMPONENT": RefImpl {
          "__v_isRef": true,
          "__v_isShallow": false,
          "_rawValue": {},
          "_value": {},
          "dep": undefined,
        },
      },
      "directives": null,
      "ec": null,
      "effect": ReactiveEffect {
        "active": true,
        "allowRecurse": true,
        "deps": [
          Set {
            [Circular],
          },
          Set {
            [Circular],
          },
          Set {
            [Circular],
          },
        ],
        "fn": [Function],
        "onTrack": undefined,
        "onTrigger": undefined,
        "parent": undefined,
        "scheduler": [Function],
      },
      "emit": [Function],
      "emitsOptions": null,
      "emitted": null,
      "exposeProxy": null,
      "exposed": null,
      "inheritAttrs": undefined,
      "isDeactivated": false,
      "isMounted": true,
      "isUnmounted": false,
      "m": null,
      "next": null,
      "parent": null,
      "props": {},
      "propsDefaults": {},
      "propsOptions": [
        {},
        [],
      ],
      "provides": {},
      "proxy": {
        "VTU_COMPONENT": {},
      },
      "refs": {
        "VTU_COMPONENT": {},
      },
      "render": [Function],
      "renderCache": [],
      "root": [Circular],
      "rtc": null,
      "rtg": null,
      "scope": EffectScope {
        "active": true,
        "cleanups": [],
        "detached": true,
        "effects": [
          ReactiveEffect {
            "active": true,
            "allowRecurse": true,
            "deps": [
              Set {
                [Circular],
              },
              Set {
                [Circular],
              },
              Set {
                [Circular],
              },
            ],
            "fn": [Function],
            "onTrack": undefined,
            "onTrigger": undefined,
            "parent": undefined,
            "scheduler": [Function],
          },
        ],
        "parent": undefined,
      },
      "setupContext": null,
      "setupState": {
        "VTU_COMPONENT": {},
      },
      "slots": {},
      "sp": null,
      "subTree": {
        "__v_isVNode": true,
        "__v_skip": true,
        "anchor": null,
        "appContext": null,
        "children": null,
        "component": {
          "a": null,
          "accessCache": {
            "@@__IMMUTABLE_ITERABLE__@@": 0,
            "@@__IMMUTABLE_RECORD__@@": 0,
            "__v_isReadonly": 0,
            "__v_isRef": 0,
            "__v_isShallow": 0,
            "__v_raw": 0,
            "__v_skip": 4,
            "_isMockFunction": 0,
            "constructor": 0,
            "toJSON": 0,
            Symbol(Symbol.toStringTag): 0,
          },
          "appContext": {
            "app": [Circular],
            "components": {
              "transition": {
                "name": "transition",
              },
              "transition-group": {
                "name": "transition-group",
              },
            },
            "config": {
              "compilerOptions": {},
              "errorHandler": undefined,
              "globalProperties": {},
              "isNativeTag": [Function],
              "optionMergeStrategies": {},
              "performance": false,
              "warnHandler": undefined,
            },
            "directives": {},
            "emitsCache": WeakMap {},
            "mixins": [
              {
                "beforeCreate": [Function],
              },
            ],
            "optionsCache": WeakMap {},
            "propsCache": WeakMap {},
            "provides": {},
            "reload": [Function],
          },
          "asyncDep": null,
          "asyncResolved": false,
          "attrs": {},
          "bc": null,
          "bm": null,
          "bu": null,
          "bum": null,
          "c": null,
          "components": {},
          "ctx": {},
          "da": null,
          "data": {},
          "directives": null,
          "ec": null,
          "effect": ReactiveEffect {
            "active": true,
            "allowRecurse": true,
            "deps": [],
            "fn": [Function],
            "onTrack": undefined,
            "onTrigger": undefined,
            "parent": undefined,
            "scheduler": [Function],
          },
          "emit": [Function],
          "emitsOptions": null,
          "emitted": null,
          "exposeProxy": null,
          "exposed": null,
          "inheritAttrs": undefined,
          "isDeactivated": false,
          "isMounted": true,
          "isUnmounted": false,
          "m": null,
          "next": null,
          "parent": [Circular],
          "props": {},
          "propsDefaults": {},
          "propsOptions": [
            {},
            [],
          ],
          "provides": {},
          "proxy": {},
          "refs": {},
          "render": [Function],
          "renderCache": [],
          "root": [Circular],
          "rtc": null,
          "rtg": null,
          "scope": EffectScope {
            "active": true,
            "cleanups": [],
            "detached": true,
            "effects": [
              ReactiveEffect {
                "active": true,
                "allowRecurse": true,
                "deps": [],
                "fn": [Function],
                "onTrack": undefined,
                "onTrigger": undefined,
                "parent": undefined,
                "scheduler": [Function],
              },
            ],
            "parent": undefined,
          },
          "setupContext": null,
          "setupState": {},
          "slots": {},
          "sp": null,
          "subTree": {
            "__v_isVNode": true,
            "__v_skip": true,
            "anchor": null,
            "appContext": null,
            "children": [
              {
                "__v_isVNode": true,
                "__v_skip": true,
                "anchor": null,
                "appContext": null,
                "children": [
                  {
                    "__v_isVNode": true,
                    "__v_skip": true,
                    "anchor": null,
                    "appContext": null,
                    "children": "Gabriel Caiana",
                    "component": null,
                    "ctx": [Circular],
                    "dirs": null,
                    "dynamicChildren": null,
                    "dynamicProps": null,
                    "el": Gabriel Caiana,
                    "key": null,
                    "patchFlag": 0,
                    "props": null,
                    "ref": null,
                    "scopeId": null,
                    "shapeFlag": 8,
                    "slotScopeIds": null,
                    "ssContent": null,
                    "ssFallback": null,
                    "staticCount": 0,
                    "suspense": null,
                    "target": null,
                    "targetAnchor": null,
                    "transition": null,
                    "type": Symbol(Text),
                  },
                ],
                "component": null,
                "ctx": [Circular],
                "dirs": null,
                "dynamicChildren": null,
                "dynamicProps": null,
                "el": <nuxt-link
                  class="font-medium"
                  to="/"
                >
                  Gabriel Caiana
                </nuxt-link>,
                "key": null,
                "patchFlag": 0,
                "props": {
                  "class": "font-medium",
                  "to": "/",
                },
                "ref": null,
                "scopeId": null,
                "shapeFlag": 17,
                "slotScopeIds": null,
                "ssContent": null,
                "ssFallback": null,
                "staticCount": 0,
                "suspense": null,
                "target": null,
                "targetAnchor": null,
                "transition": null,
                "type": "nuxt-link",
              },
              {
                "__v_isVNode": true,
                "__v_skip": true,
                "anchor": null,
                "appContext": null,
                "children": [
                  {
                    "__v_isVNode": true,
                    "__v_skip": true,
                    "anchor": null,
                    "appContext": null,
                    "children": [
                      {
                        "__v_isVNode": true,
                        "__v_skip": true,
                        "anchor": null,
                        "appContext": null,
                        "children": null,
                        "component": null,
                        "ctx": [Circular],
                        "dirs": null,
                        "dynamicChildren": null,
                        "dynamicProps": null,
                        "el": <icon
                          name="ph:github-logo-fill"
                          size="24"
                        />,
                        "key": null,
                        "patchFlag": 0,
                        "props": {
                          "name": "ph:github-logo-fill",
                          "size": "24",
                        },
                        "ref": null,
                        "scopeId": null,
                        "shapeFlag": 1,
                        "slotScopeIds": null,
                        "ssContent": null,
                        "ssFallback": null,
                        "staticCount": 0,
                        "suspense": null,
                        "target": null,
                        "targetAnchor": null,
                        "transition": null,
                        "type": "Icon",
                      },
                    ],
                    "component": null,
                    "ctx": [Circular],
                    "dirs": null,
                    "dynamicChildren": null,
                    "dynamicProps": null,
                    "el": <a
                      href="https://github.com/gabrielcaiana"
                      target="_blank"
                    >
                      <icon
                        name="ph:github-logo-fill"
                        size="24"
                      />
                    </a>,
                    "key": null,
                    "patchFlag": 0,
                    "props": {
                      "href": "https://github.com/gabrielcaiana",
                      "target": "_blank",
                    },
                    "ref": null,
                    "scopeId": null,
                    "shapeFlag": 17,
                    "slotScopeIds": null,
                    "ssContent": null,
                    "ssFallback": null,
                    "staticCount": 0,
                    "suspense": null,
                    "target": null,
                    "targetAnchor": null,
                    "transition": null,
                    "type": "a",
                  },
                  {
                    "__v_isVNode": true,
                    "__v_skip": true,
                    "anchor": null,
                    "appContext": null,
                    "children": [
                      {
                        "__v_isVNode": true,
                        "__v_skip": true,
                        "anchor": null,
                        "appContext": null,
                        "children": null,
                        "component": null,
                        "ctx": [Circular],
                        "dirs": null,
                        "dynamicChildren": null,
                        "dynamicProps": null,
                        "el": <icon
                          name="ph:twitter-logo-fill"
                          size="24"
                        />,
                        "key": null,
                        "patchFlag": 0,
                        "props": {
                          "name": "ph:twitter-logo-fill",
                          "size": "24",
                        },
                        "ref": null,
                        "scopeId": null,
                        "shapeFlag": 1,
                        "slotScopeIds": null,
                        "ssContent": null,
                        "ssFallback": null,
                        "staticCount": 0,
                        "suspense": null,
                        "target": null,
                        "targetAnchor": null,
                        "transition": null,
                        "type": "Icon",
                      },
                    ],
                    "component": null,
                    "ctx": [Circular],
                    "dirs": null,
                    "dynamicChildren": null,
                    "dynamicProps": null,
                    "el": <a
                      href="https://twitter.com/gabriel_caiana"
                      target="_blank"
                    >
                      <icon
                        name="ph:twitter-logo-fill"
                        size="24"
                      />
                    </a>,
                    "key": null,
                    "patchFlag": 0,
                    "props": {
                      "href": "https://twitter.com/gabriel_caiana",
                      "target": "_blank",
                    },
                    "ref": null,
                    "scopeId": null,
                    "shapeFlag": 17,
                    "slotScopeIds": null,
                    "ssContent": null,
                    "ssFallback": null,
                    "staticCount": 0,
                    "suspense": null,
                    "target": null,
                    "targetAnchor": null,
                    "transition": null,
                    "type": "a",
                  },
                  {
                    "__v_isVNode": true,
                    "__v_skip": true,
                    "anchor": null,
                    "appContext": null,
                    "children": [
                      {
                        "__v_isVNode": true,
                        "__v_skip": true,
                        "anchor": null,
                        "appContext": null,
                        "children": null,
                        "component": null,
                        "ctx": [Circular],
                        "dirs": null,
                        "dynamicChildren": null,
                        "dynamicProps": null,
                        "el": <icon
                          name="ph:linkedin-logo-fill"
                          size="24"
                        />,
                        "key": null,
                        "patchFlag": 0,
                        "props": {
                          "name": "ph:linkedin-logo-fill",
                          "size": "24",
                        },
                        "ref": null,
                        "scopeId": null,
                        "shapeFlag": 1,
                        "slotScopeIds": null,
                        "ssContent": null,
                        "ssFallback": null,
                        "staticCount": 0,
                        "suspense": null,
                        "target": null,
                        "targetAnchor": null,
                        "transition": null,
                        "type": "Icon",
                      },
                    ],
                    "component": null,
                    "ctx": [Circular],
                    "dirs": null,
                    "dynamicChildren": null,
                    "dynamicProps": null,
                    "el": <a
                      href="https://www.linkedin.com/in/gabrielcaiana/"
                      target="_blank"
                    >
                      <icon
                        name="ph:linkedin-logo-fill"
                        size="24"
                      />
                    </a>,
                    "key": null,
                    "patchFlag": 0,
                    "props": {
                      "href": "https://www.linkedin.com/in/gabrielcaiana/",
                      "target": "_blank",
                    },
                    "ref": null,
                    "scopeId": null,
                    "shapeFlag": 17,
                    "slotScopeIds": null,
                    "ssContent": null,
                    "ssFallback": null,
                    "staticCount": 0,
                    "suspense": null,
                    "target": null,
                    "targetAnchor": null,
                    "transition": null,
                    "type": "a",
                  },
                ],
                "component": null,
                "ctx": [Circular],
                "dirs": null,
                "dynamicChildren": null,
                "dynamicProps": null,
                "el": <div
                  class="flex gap-3"
                >
                  <a
                    href="https://github.com/gabrielcaiana"
                    target="_blank"
                  >
                    <icon
                      name="ph:github-logo-fill"
                      size="24"
                    />
                  </a>
                  <a
                    href="https://twitter.com/gabriel_caiana"
                    target="_blank"
                  >
                    <icon
                      name="ph:twitter-logo-fill"
                      size="24"
                    />
                  </a>
                  <a
                    href="https://www.linkedin.com/in/gabrielcaiana/"
                    target="_blank"
                  >
                    <icon
                      name="ph:linkedin-logo-fill"
                      size="24"
                    />
                  </a>
                </div>,
                "key": null,
                "patchFlag": 0,
                "props": {
                  "class": "flex gap-3",
                },
                "ref": null,
                "scopeId": null,
                "shapeFlag": 17,
                "slotScopeIds": null,
                "ssContent": null,
                "ssFallback": null,
                "staticCount": 0,
                "suspense": null,
                "target": null,
                "targetAnchor": null,
                "transition": null,
                "type": "div",
              },
            ],
            "component": null,
            "ctx": [Circular],
            "dirs": null,
            "dynamicChildren": [],
            "dynamicProps": null,
            "el": <header
              class="pt-8 pb-4 px-6 text-white border-b border-neutral-800 flex justify-between items-center"
            >
              <nuxt-link
                class="font-medium"
                to="/"
              >
                Gabriel Caiana
              </nuxt-link>
              <div
                class="flex gap-3"
              >
                <a
                  href="https://github.com/gabrielcaiana"
                  target="_blank"
                >
                  <icon
                    name="ph:github-logo-fill"
                    size="24"
                  />
                </a>
                <a
                  href="https://twitter.com/gabriel_caiana"
                  target="_blank"
                >
                  <icon
                    name="ph:twitter-logo-fill"
                    size="24"
                  />
                </a>
                <a
                  href="https://www.linkedin.com/in/gabrielcaiana/"
                  target="_blank"
                >
                  <icon
                    name="ph:linkedin-logo-fill"
                    size="24"
                  />
                </a>
              </div>
            </header>,
            "key": null,
            "patchFlag": 0,
            "props": {
              "class": "pt-8 pb-4 px-6 text-white border-b border-neutral-800 flex justify-between items-center",
            },
            "ref": null,
            "scopeId": null,
            "shapeFlag": 17,
            "slotScopeIds": null,
            "ssContent": null,
            "ssFallback": null,
            "staticCount": 0,
            "suspense": null,
            "target": null,
            "targetAnchor": null,
            "transition": null,
            "type": "header",
          },
          "suspense": null,
          "suspenseId": 0,
          "type": {
            "__file": "/Users/gabrielcaiana/www/Frontend/nuxt3/website/components/header/index.vue",
            "components": {},
            "render": [Function],
          },
          "u": null,
          "uid": 1,
          "um": null,
          "update": [Function],
          "vnode": [Circular],
          "withProxy": null,
        },
        "ctx": [Circular],
        "dirs": null,
        "dynamicChildren": null,
        "dynamicProps": null,
        "el": <header
          class="pt-8 pb-4 px-6 text-white border-b border-neutral-800 flex justify-between items-center"
        >
          <nuxt-link
            class="font-medium"
            to="/"
          >
            Gabriel Caiana
          </nuxt-link>
          <div
            class="flex gap-3"
          >
            <a
              href="https://github.com/gabrielcaiana"
              target="_blank"
            >
              <icon
                name="ph:github-logo-fill"
                size="24"
              />
            </a>
            <a
              href="https://twitter.com/gabriel_caiana"
              target="_blank"
            >
              <icon
                name="ph:twitter-logo-fill"
                size="24"
              />
            </a>
            <a
              href="https://www.linkedin.com/in/gabrielcaiana/"
              target="_blank"
            >
              <icon
                name="ph:linkedin-logo-fill"
                size="24"
              />
            </a>
          </div>
        </header>,
        "key": null,
        "patchFlag": 0,
        "props": {
          "ref": "VTU_COMPONENT",
        },
        "ref": {
          "f": false,
          "i": [Circular],
          "k": undefined,
          "r": "VTU_COMPONENT",
        },
        "scopeId": null,
        "shapeFlag": 4,
        "slotScopeIds": null,
        "ssContent": null,
        "ssFallback": null,
        "staticCount": 0,
        "suspense": null,
        "target": null,
        "targetAnchor": null,
        "transition": null,
        "type": {
          "__file": "/Users/gabrielcaiana/www/Frontend/nuxt3/website/components/header/index.vue",
          "components": {},
          "render": [Function],
        },
      },
      "suspense": null,
      "suspenseId": 0,
      "type": {
        "name": "VTU_ROOT",
        "render": [Function],
        "setup": [Function],
      },
      "u": null,
      "uid": 0,
      "um": null,
      "update": [Function],
      "vnode": {
        "__v_isVNode": true,
        "__v_skip": true,
        "anchor": null,
        "appContext": {
          "app": [Circular],
          "components": {
            "transition": {
              "name": "transition",
            },
            "transition-group": {
              "name": "transition-group",
            },
          },
          "config": {
            "compilerOptions": {},
            "errorHandler": undefined,
            "globalProperties": {},
            "isNativeTag": [Function],
            "optionMergeStrategies": {},
            "performance": false,
            "warnHandler": undefined,
          },
          "directives": {},
          "emitsCache": WeakMap {},
          "mixins": [
            {
              "beforeCreate": [Function],
            },
          ],
          "optionsCache": WeakMap {},
          "propsCache": WeakMap {},
          "provides": {},
          "reload": [Function],
        },
        "children": null,
        "component": [Circular],
        "ctx": null,
        "dirs": null,
        "dynamicChildren": null,
        "dynamicProps": null,
        "el": <header
          class="pt-8 pb-4 px-6 text-white border-b border-neutral-800 flex justify-between items-center"
        >
          <nuxt-link
            class="font-medium"
            to="/"
          >
            Gabriel Caiana
          </nuxt-link>
          <div
            class="flex gap-3"
          >
            <a
              href="https://github.com/gabrielcaiana"
              target="_blank"
            >
              <icon
                name="ph:github-logo-fill"
                size="24"
              />
            </a>
            <a
              href="https://twitter.com/gabriel_caiana"
              target="_blank"
            >
              <icon
                name="ph:twitter-logo-fill"
                size="24"
              />
            </a>
            <a
              href="https://www.linkedin.com/in/gabrielcaiana/"
              target="_blank"
            >
              <icon
                name="ph:linkedin-logo-fill"
                size="24"
              />
            </a>
          </div>
        </header>,
        "key": null,
        "patchFlag": 0,
        "props": null,
        "ref": null,
        "scopeId": null,
        "shapeFlag": 4,
        "slotScopeIds": null,
        "ssContent": null,
        "ssFallback": null,
        "staticCount": 0,
        "suspense": null,
        "target": null,
        "targetAnchor": null,
        "transition": null,
        "type": {
          "name": "VTU_ROOT",
          "render": [Function],
          "setup": [Function],
        },
      },
      "withProxy": null,
    },
    "_props": null,
    "_uid": 0,
    "component": [Function],
    "config": {
      "compilerOptions": {},
      "errorHandler": undefined,
      "globalProperties": {},
      "isNativeTag": [Function],
      "optionMergeStrategies": {},
      "performance": false,
      "warnHandler": undefined,
    },
    "directive": [Function],
    "mixin": [Function],
    "mount": [Function],
    "provide": [Function],
    "unmount": [Function],
    "use": [Function],
    "version": "3.2.45",
  },
  "__setProps": [Function],
  "cleanUpCallbacks": [
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
  ],
  "componentVM": {},
  "isDisabled": [Function],
  "rootVM": {
    "VTU_COMPONENT": {},
  },
  "wrapperElement": <header
    class="pt-8 pb-4 px-6 text-white border-b border-neutral-800 flex justify-between items-center"
  >
    <nuxt-link
      class="font-medium"
      to="/"
    >
      Gabriel Caiana
    </nuxt-link>
    <div
      class="flex gap-3"
    >
      <a
        href="https://github.com/gabrielcaiana"
        target="_blank"
      >
        <icon
          name="ph:github-logo-fill"
          size="24"
        />
      </a>
      <a
        href="https://twitter.com/gabriel_caiana"
        target="_blank"
      >
        <icon
          name="ph:twitter-logo-fill"
          size="24"
        />
      </a>
      <a
        href="https://www.linkedin.com/in/gabrielcaiana/"
        target="_blank"
      >
        <icon
          name="ph:linkedin-logo-fill"
          size="24"
        />
      </a>
    </div>
  </header>,
}
`;
